packages/jest-preset-default/scripts/pegjs-transform.js

```diff
@@ -1,16 +0,0 @@
-const pegjs = require( 'pegjs' );
```

Comment:
We don't need it outside of Gutenberg.

Reply:
Could this have a breaking effect that needs to be documented if someone is using @wordpress/jest-preset-default ?

Reply:
We should include it in the changelog. We can also bump version to 2.0.0 to follow the semantic versioning. I don't think that other code that `blocks` module in Gutenberg would ever need to use it.

Reply:
@nerrad, can you confirm it can be removed or should we keep it as long as `blocks` depend on it?

Reply:
I think this babel7 change needs a bump to `2.0.0` anyway. I'm wondering if we should update the pragma at the same time to avoid the need for a `3.0.0` later

Reply:
Hmm, we can flip the pragma handling in Gutenberg, good idea 👍

Reply:
I still don't totally grok what pegjs is used for (Does it provide the grammar rules for the custom html comments for blocks?).  Assuming that's what its for, and it only is required by GB `blocks` - plugins writing tests can probably just mock any GB blocks in their tests.